pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -384,6 +393,19 @@ func (h *NetworkUtilsHandler) StartDHCP(nic *VIF, serverAddr net.IP, bridgeInter
 	return nil
 }
 
+func (h *NetworkUtilsHandler) CreateNDPConnection(bridgeInterfaceName string, launcherPID int) error {
+	log.Log.Infof("Starting RA daemon on network Nic: %s", bridgeInterfaceName)
+	ndpConnection, err := ndp.NewNDPConnection(bridgeInterfaceName)
+	if err != nil {
+		return fmt.Errorf("failed to create the RouterAdvertisement daemon: %v", err)
+	}
+
+	return ndpConnection.Export(
```

Comment:
Please add to the commit message that a follow-up commit will move the `nfd.Export` invocation to a separate go routine.